taxi/models.py

```diff
@@ -15,7 +16,21 @@ def __str__(self):
 
 
 class Driver(AbstractUser):
-    license_number = models.CharField(max_length=255, unique=True)
+    license_number = models.CharField(
+        max_length=8,
+        unique=True,
+        validators=[
+            validators.RegexValidator(
+                regex="^[A-Z]{3}\\d{5}$",
+                message=(
+                    "Your license number should "
+                    "start with 3 uppercase letters "
+                    "and end with 5 digits."
+                ),
+                code="wrong license number"
```

Comment:
Move this check to view validation via form, since you won't be able to create custom user license via admin panel. Also better not to use regex since it's not intuitive and explicit also, sometimes, pretty slow.